engine/table/src/main/java/io/deephaven/engine/util/PyCallableWrapperJpyImpl.java

```diff
@@ -105,9 +118,25 @@ private void prepareSignature() {
                                 + " has multiple signatures; this is not currently supported for numba vectorized functions");
             }
             signature = params.get(0).getStringValue();
-            unwrapped = null;
+            unwrapped = pyCallable;
             numbaVectorized = true;
             vectorized = true;
+        } else if (pyCallable.equals(NUMBA_GUVECTORIZED_FUNC_TYPE)) {
+            List<PyObject> params = pyCallable.getAttribute("types").asList();
+            if (params.isEmpty()) {
+                throw new IllegalArgumentException(
+                        "numba guvectorized function must have an explicit signature: " + pyCallable);
+            }
+            // numba allows a vectorized function to have multiple signatures
+            if (params.size() > 1) {
+                throw new UnsupportedOperationException(
+                        pyCallable
+                                + " has multiple signatures; this is not currently supported for numba vectorized functions");
+            }
+            signature = params.get(0).getStringValue();
+            unwrapped = pyCallable;
+            numbaVectorized = false;
```

Comment:
I think possibly `numbaVectorized = true`.

See https://numba.pydata.org/numba-doc/dev/user/vectorize.html#the-guvectorize-decorator.
Look at the example that begins with `The nice thing is that NumPy will automatically dispatch over more complicated inputs, depending on their shapes:`.  In the example, the guvectorize function is `(n),()->(n)`, but it is applied to arrays of other shapes in a vectorized way.

Reply:
It would be nice to support that. But the vectorization implementation currently doesn't support non scalar types.

Reply:
Should this become a ticket and item on our project plan so that we don't try to tackle it in this PR?